packages/sdk/src/restclient.ts

```diff
@@ -22,25 +22,47 @@ interface NodeInfoResponse {
   readonly node_info: NodeInfo;
 }
 
-interface BlockMeta {
-  readonly header: {
-    readonly height: number;
-    readonly time: string;
-    readonly num_txs: number;
-  };
-  readonly block_id: {
-    readonly hash: string;
-  };
+export interface BlockHeader {
```

Comment:
Adding full info is good

Reply:
It seems like this changed since cosmos hub (gaia 2)?

It would be nice to make sure the test hit all the fields to make sure they are there (and quickly break if we upgrade to an incompatible version)

Reply:
Yeah seems like that. I added tests for all the fields in the types.